app/models/Post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_create :update_post_counter
+
+  def update_post_counter
+    author.update(posts_counter: author.posts.size)
+  end
```

Comment:
## Optional
- [ ] This method should be private after adding [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save`